### PR TITLE
Authgear

### PR DESCRIPTION
**Pull request by louischan-oursky**

### CODE REVIEW

**Review by kiootic — COMMENTED** [review]
I remember from the previous discussion that, we can just start a new repo and rewrite the SDK, since so much would need to be changed?

**Comment by louischan-oursky** [comment]
I want to keep the history. Starting from scratch has some overhead.

**Review by kiootic — COMMENTED** [review]
We may as well delete `.travis.yaml` and `CHANGELOG`?

**Comment by louischan-oursky** [comment]
updated.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: "12"
+    - run: npm ci
+    - run: npm run lerna bootstrap
+    - run: npm run format
+      if: ${{ always() }}
```
Comment (kiootic) [inline .github/workflows/ci.yaml]: I suppose `!cancelled()` would be better.

Comment (louischan-oursky) [reply .github/workflows/ci.yaml]: updated.

**Review by kiootic — APPROVED** [review]

**Pull request merged**